commander/src/bootstrapping/commands/blockchain/export.ts

```diff
@@ -37,19 +37,19 @@ export class ExportCommand extends Command {
 		const dataPath = flags['data-path']
 			? flags['data-path']
 			: getDefaultPath(this.config.pjson.name);
-		const blockchainPath = getBlockchainDBPath(dataPath);
+		const inputPath = join(dataPath, 'data');
 		const exportPath = flags.output ? flags.output : process.cwd();
 
 		this.log('Exporting blockchain:');
-		this.log(`   ${getFullPath(blockchainPath)}`);
+		this.log(`   ${getFullPath(inputPath)}`);
 		const filePath = join(exportPath, 'blockchain.db.tar.gz');
```

Comment:
Since now it contains multiple db, maybe it's better to name it `backup.tar.gz` or something similar.
Also, if the filePath doesn't exist it fails, so better to create folder if it doesn't exist

Reply:
I think the name of the file should be aligned with the logs `Exporting blockchain` and the command name itself so I think it's okay to not change or otherwise rename to `blockchain.tar.gz` whichever you prefer.

What do you mean it fails ? If no `--output-path` flag is provided it outputs to `process.cwd()`.

Reply:
Make sense, then I think `blockchain.tar.gz` as you mentioned would be better

it fails when `--output-path` is specified and the target folder doesn't exist.